### DISCUSSION TITLE
Rough estimate of next stable release date?

### DISCUSSION THREAD

**Discussion by Nekkowe** [cboxdoerfer/fsearch#508] [Q&A] [answered]
Hello! Thank you for this incredible program.
I'd been looking for a Linux alternative to Everything, and this hits the mark incredibly well.

The only things I miss are a preview pane for images (apparently already planned) and image thumbnails - apparently, a pull request for that last one got merged recently, but it hasn't made its way to the stable release yet.

Are there any plans for when, roughly speaking, the next stable release will be deployed?
Should I switch to the development branch for now rather than wait?

**Comment by cboxdoerfer** [comment, marked as answer]
Hi, there a few show-stoppers in the development for the next release. They're mostly race conditions which crash the application and are difficult to reproduce and debug. Hence it's difficult to estimate when they'll be fixed.

The master branch however (which the published development builds are based on) is actually pretty close to the last release. The major difference are the image thumbnails and previews. So the development branch is quite stable and I'll try to keep it that way.
